apps/ui/postcss.config.js

```diff
@@ -1,4 +1,4 @@
-module.exports = {
+export default {
```

Comment:
Shouldn't we change the extension of this file to postcss.config.ts ?

Reply:
No, it's just using ESM which supports this syntax. Node will interpret this file as ESM because we added `type: "module"` in package.json (so it doesn't need `.mjs` extension either).